### ISSUE TITLE
check_dependencies throws exception due to missing requirements.txt

### ISSUE THREAD

**Issue by fidgetingbits** [pwndbg/pwndbg#1865]
### Description

When an exception is triggered and you have `set exception-verbose on` the pwndbng exception handler will call a function called `check_dependencies` in `pwndbg/exception.py`, which will do the following:

```
    project_path = os.path.dirname(os.path.abspath(__file__))
    requirements_path = os.path.join(project_path, os.pardir, "requirements.txt")
    with open(requirements_path, "r") as f:
```

It seems like the `requirements.txt` file is only present in the `docs/` folder atm, so this triggers its own exception. I confirmed that just symlinking `requirements.txt` to the real file fixes the issue. 

### Steps to reproduce

Run `set exception-verbose on` and cause pwndbg to trigger some python exception.

### My setup

I am using the latest dev branch and am not running in the virtual environment

**Comment by disconnect3d** [comment]
We moved dependencies versions to https://github.com/pwndbg/pwndbg/blob/3d6f86402ea01c69199f0f60ed1f88bc4ddca308/pyproject.toml#L261-L273 

We need to fetch them differently here

**Comment by dmur1** [comment]
will take a look at this

**Comment by dmur1** [comment]
@disconnect3d https://github.com/pwndbg/pwndbg/pull/1880

**Comment by disconnect3d** [comment]
Fixed in #1880. Let's skip check_deps for now... we can bring back this issue if someone reports _yet another issue_ that is about outdated dependencies.
